package/yast2-schema.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Apr 16 15:05:59 UTC 2020 - Josef Reidinger <jreidinger@suse.com>
+
+- Always add to schema registration (jcs#SLE-11852)
```

Comment:
```suggestion
- Always add to schema registration (jsc#SLE-11852)
```

Reply:
Also:

Correct:
> Always add the registration to the schema

Acceptable:
> Always add registration to schema

Means "Always add (what?) to registration of schema:
> ~Always add to schema registration~